src/main/java/com/rmn/qa/aws/AwsVmManager.java

```diff
@@ -493,8 +493,11 @@ String getNodeConfig(final String uuid, final String hostName, final String brow
         Date createdDate = Calendar.getInstance().getTime();
 
         // Pass in the created date so we can know when this node was spun up
-        nodeConfig = nodeConfig.replaceAll("<CREATED_DATE>", AwsVmManager.NODE_DATE_FORMAT.format(createdDate));
-        nodeConfig = nodeConfig.replaceFirst("<HOST_NAME>", hostName);
+        String nodeCreatedDate = AwsVmManager.NODE_DATE_FORMAT.format(createdDate);
```

Comment:
when I tested this, the the node was created, the createdDate in nodeConfig.json had date like createdDate=05/25/2017 09:46:05 , but the node wouldn't start with json parsing error as selenium webdriver 3.4 I believe due to selenium using Jcommander. Once I put the dates between "" this started working fine.

Another note, current format of command  in node bootstrap code wouldn't start the node. All -D parameters should come before -jar option 
see here https://github.com/SeleniumHQ/selenium/issues/2566